jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -10,10 +10,10 @@ version:
 :toc: macro
 :toc-title:
 
-Migrating from 2.19.x to 3.0 requires that you backup your 2.19 installation and then restore the data to the new Server 3.0 installation.
-It does require an already operating Server 3.0 installation. Depending on the size of your data stores, the migration
-can take anywhere from a few minutes to a few hours. We recommend using a staging environment before completing this
-process in a production environment. This will not only allow you to gain a better understanding of the migration process,
+Migrating from 2.19.x to 3.0 requires you to back up your 2.19 instance data (Mongo, Postgres, and Vault) and then restore that data in a waiting Server 3.0
+instance. If you run into trouble you can fallback to your 2.19 instance. It does require an already operating Server 3.0 installation. Depending 
```

Comment:
```suggestion
instance. If you run into trouble you can fallback to your 2.19 instance, in this case your 2.19 installation will need to be restarted. Before starting the migration process you will need to have an operational Server 3.x installation up and running. Depending 
```

Reply:
@nathanfishcircleci could you check if this works?

Reply:
Customers can start the application with `replicatedctl app start` or from the UI of the management console.